storage/innobase/fil/fil0fil.cc

```diff
@@ -2605,7 +2605,7 @@ fil_ibd_load(uint32_t space_id, const char *filename, fil_space_t *&space)
 
 	if (space) {
 		sql_print_information("InnoDB: Ignoring data file '%s'"
-				      " with space ID " ULINTPF
+				      " with space ID %" PRIu32
 				      ". Another data file called %s"
 				      " exists"
 				      " with the same space ID.",
```

Comment:
All fixes like this must be submitted separately, targeting the 10.11 branch and marked as a fixup of ca501ffb04246dcaa1f1d433d916d8436e30602e (MDEV-26195).

Once that fix has been reviewed and merged to 10.11, I’d like to see another pull request that combines ca501ffb04246dcaa1f1d433d916d8436e30602e and all follow-up fixes and targets the 10.6 branch. This is because it is possible that the 10.6 branch will remain in support for quite some time, and we want to avoid any surprises when merging changes from 10.6 to later branches, or when porting changes from later branches to 10.6.

Reply:
> All fixes like this must be submitted separately, targeting the 10.11 branch and marked as a fixup of ca501ffb04 (MDEV-26195).

I had this fix extracted as a part of #3518.
`bnestere` requested per-version logs on JIRA; I’ll mention available relevant tickets/commits as I prepare them.

> Once that fix has been reviewed and merged to 10.11, I’d like to see another pull request that combines ca501ffb04 and all follow-up fixes and targets the 10.6 branch.

[MDEV-26195](https://jira.mariadb.org/browse/MDEV-26195) desn’t have any “Fix Version/s” earlier than `10.7.0`. Would you like it backported to 10.6?

> we want to avoid any surprises when merging changes from 10.6 to later branches, or when porting changes from later branches to 10.6.

[We fix bugs on the earliest (supported) branch and forwardport by merging.](https://github.com/MariaDB/server/blob/https://github.com/MariaDB/server/commit/e6df06d40d/CODING_STANDARDS.md#target-branch)
Do we occasionally also backport (besides custom builds)?